bundles/org.openhab.binding.tacmi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -173,4 +173,50 @@
 		<state readOnly="true"/>
 		<config-description-ref uri="channel-type:tacmi:schemaApiDefaults"/>
 	</channel-type>
+
+
+	<!-- JSON -->
+
+
+	<thing-type id="cmiJSON" extensible="schema-switch-ro,schema-numeric-ro,schema-state-ro">
+		<label>TA C.M.I. JSON API Connection</label>
+		<description>Communication via the "JSONAPI" on a "Technische Alternative C.M.I. Control and Monitoring
+			Interface"</description>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>C.M.I. IP Address</label>
+				<description>Host name or IP address of the C.M.I.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="username" type="text" required="true">
+				<label>User name</label>
+				<description>User name for authentication on the C.M.I.</description>
+			</parameter>
+			<parameter name="password" type="text">
+				<label>Password</label>
+				<description>Password for authentication on the C.M.I.</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="nodeId" type="integer" required="true" min="0">
+				<label>Node Id</label>
+				<description>The node ID of the device you want to monitor (CMI → CAN-Bus)</description>
+			</parameter>
+			<parameter name="params" type="text">
+				<label>API-Parameters</label>
+				<description><![CDATA[ The params to query. E.g. I,O,Sg (See the <a href="https://wiki.ta.co.at/C.M.I._JSON-API">API documentation</a> for details)<br />
+					Possible options are: Inputs (I), Outputs (O), General (Sg), Logging Analog (La), Logging Digital (Ld) ]]>
+				</description>
+				<default>I,O,Sg</default>
+			</parameter>
+			<parameter name="pollInterval" type="integer" min="10" required="true">
```

Comment:
As you provide a default value, no need to mark the parameter as required.

Reply:
But it mustn't be left empty. As I understood required forces the user to insert something, no?